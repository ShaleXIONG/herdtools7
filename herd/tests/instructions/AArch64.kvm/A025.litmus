AArch64 A025
Variant=vmsa
{
[PTE(x)]=(oa:PA(x),valid:1,dbm:0,db:0);
0:X1=x;
0:X2=0;
}
 P0             ;
L0:             ;
 AT S1E0W, X1   ;
 ISB            ;
 MRS X3, PAR_EL1;
MOV W2,#1       ;
forall(0:X3=parel1_t:(f:1) /\ 0:X2=1)