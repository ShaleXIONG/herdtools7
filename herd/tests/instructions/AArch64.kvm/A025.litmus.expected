Test A025 Required
States 1
0:X2=1; 0:X3=(parel1_t:(f:1));
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X3=(parel1_t:(f:1)) /\ 0:X2=1)
Observation A025 Always 1 0
Hash=1fda2621ce7823df29288b9085d81031

